src/lib/dateTime.ts

```diff
@@ -39,4 +54,12 @@ export class DateTime {
 	static yearMonthDayToDate(year: string, month: string, day: string) {
 		return moment(`${year.padStart(4, '0')}${month.padStart(2, '0')}${day.padStart(2, '0')}`)
 	}
+
+	static sortDateRanges(dateRange1: DateRange, dateRange2: DateRange) {
+		if (dateRange1.date > dateRange2.date) {
+			return 1
+		} else {
+			return -1
+		}
```

Comment:
a ternary operator or even
```suggestion
		return +(dateRange1.date > dateRange2.date) || -1
```